.github/workflows/helm-test.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
           python-version: "3.10"
 
       - name: Set up chart-testing
-        uses: helm/chart-testing-action@e8788873172cb653a90ca2e819d79d65a66d4e76 # v2.4.0
+        uses: helm/chart-testing-action@e6669bcd63d7cb57cb4380c33043eebe5d111992 # v2.6.1
```

Comment:
Will be removed once https://github.com/weaveworks/tf-controller/pull/1147 is merged.